.vscode/echos.code-snippets

```diff
@@ -59,6 +59,22 @@
         ],
         "body": "echo_info \"${1:message}\""
     },
+    "Echo error": {
+        "scope": "shellscript",
+        "prefix": [
+            "echo_error",
+            "error"
+        ],
+        "body": "echo_error \"${1:message}\""
+    },
+    "Echo warn": {
+        "scope": "shellscript",
+        "prefix": [
+            "echo_warn",
+            "warn"
+        ],
+        "body": "echo_warn \"${1:message}\""
+    },
```

Comment:
There is a separate PR for these, yes?

Reply:
That already got merged in elsewhere so I'll revert these